### ISSUE TITLE
Possibly Incorrect return value of `generate_sequences`

### ISSUE THREAD

**Issue by tsunrise** [tsunrise/everybody-compose#1]
https://github.com/tsunrise/cs230-proj/blob/7dd5fb490c172b69aca9ebb85b1f730e3ce3d806/preprocess/dataset.py#L68-L86

For this project, `y_notes` is not "the next expected note that the network should predict given the beat sequence". It's a sequence of notes of length is `seq_length`. Therefore, the shape of `y_notes` is not `(num_examples, 1)`, it should be something similar to `(num_examples, seq_length, num_notes)` where `num_notes` is the number of all possible notes we can expect (for piano it's ~~52~~ 88). 

In other words, suppose on the `j`'s note of `i`'s example, the note is 8, then `y_notes[i][j] = [0,0,0,0,0,0,0,1,0,0,0,0...,0]`. 

@YixinLiu030
